client/packages/system/src/Patient/DetailView/ProgramEnrolment/api/hooks/document/useProgramEnrolments.ts

```diff
@@ -1,14 +1,21 @@
-import { useQuery, useUrlQueryParams } from '@openmsupply-client/common';
+import {
+  useParams,
+  useQuery,
+  useUrlQueryParams,
+} from '@openmsupply-client/common';
 import { useProgramEnrolmentApi } from '../utils/useProgramEnrolmentApi';
 
 export const useProgramEnrolments = () => {
   const api = useProgramEnrolmentApi();
   const { queryParams } = useUrlQueryParams({
     initialSort: { key: 'type', dir: 'asc' },
   });
+  const { patientId } = useParams();
```

Comment:
Since there is already a hook defined to do this:
```suggestion
  const patientId = usePatient.utils.id();
```

after adding in 

```
import { usePatient } from '../../../../../api';
```

Reply:
thanks done